### ISSUE TITLE
Incomplete network events information in lookup_rdap()

### ISSUE THREAD

**Issue by SalmaanP** [secynic/ipwhois#188]
I was interested in finding domain registration date of certain domains and did this but it didn't have the registration timestamp in it.

```
 >>> IPWhois('77.88.55.60').lookup_rdap(depth=1)['network']['events']

[{'action': u'last changed', 'timestamp': u'2014-03-26T09:17:12Z', 'actor': None}]
```

While if i do this for another IP I do get the registration information.

```
>>> IPWhois('74.125.225.229').lookup_rdap(depth=1)['network']['events']

[{'action': u'last changed', 'timestamp': u'2012-02-24T09:44:34-05:00', 'actor': None},

 {'action': u'registration', 'timestamp': u'2007-03-13T12:09:54-04:00', 'actor': None}]

```

Any idea why this happens and how to get registration date for domains?

**Comment by secynic** [comment]
For the first IP, 'last changed' is the only event provided by RIPE:
https://rdap.db.ripe.net/ip/77.88.55.60

## Legacy Method
You can get the registration info 'created' via legacy whois.

### 'created' for smallest subnet
```python
from ipwhois import ipwhois
result = ipwhois.IPWhois('77.88.55.60').lookup_whois()
print(result['nets'][-1]['created'])
```

The result shows a bug for the value in result['nets'][1]['range']. This should be checked for CIDR and convered to 'start_ip - end_ip'.

**Comment by secynic** [comment]
Merged in #189
